packages/core/EventManager.ts

```diff
@@ -156,6 +156,42 @@ export default class EventManager {
     };
   }
 
+  public async updateLocation(event: CalendarEvent, booking: PartialBooking): Promise<CreateUpdateResult> {
+    const evt = processLocation(event);
+    const isDedicated = evt.location ? isDedicatedIntegration(evt.location) : null;
+
+    const results: Array<EventResult<Exclude<Event, AdditionalInformation>>> = [];
+    // If and only if event type is a dedicated meeting, create a dedicated video meeting.
+    if (isDedicated) {
+      const result = await this.createVideoEvent(evt);
+      if (result.createdEvent) {
+        evt.videoCallData = result.createdEvent;
+      }
+
+      results.push(result);
+    }
+
+    // Update the calendar event with the proper video call data
+    results.push(...(await this.updateAllCalendarEvents(evt, booking)));
```

Comment:
Not working for Google Meet. Hangout link is not created in updateEvent (https://github.com/calcom/cal.com/blob/fix/location-change/packages/app-store/googlecalendar/lib/CalendarService.ts)